core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -723,6 +724,8 @@ class PlaintextConsumerTest extends BaseConsumerTest {
       (record.topic, record.partition, new String(record.key), new String(record.value), record.timestamp)
     }.toSet
 
+    println("Actual " + actual.size)
```

Comment:
Is `println` the right choice here? Nothing else in these tests seem to use it? Why not normal logging?